app/assets/stylesheets/getting-started.scss

```diff
@@ -42,13 +42,16 @@
   .close { line-height: 18px; }
 }
 
+#gs-shim {
+  position: relative;
+  top: 40px;
+}
+
 #welcome-to-diaspora {
   background: orange;
   box-shadow: inset 0 -2px 10px rgba(0,0,0,0.35);
-  margin-bottom: 20px;
-  margin-top: -40px;
+  padding-top: 20px;
```

Comment:
Properties should be ordered background, box-shadow, padding-bottom, padding-top